src/main/java/io/odpf/depot/http/request/body/RawBody.java

```diff
@@ -17,11 +18,15 @@ public RawBody(HttpSinkConfig config) {
 
     @Override
     public String build(OdpfMessage message) {
-        JSONObject payload = new JSONObject();
-        payload.put("log_key", encodedSerializedStringFrom((byte[]) message.getLogKey()));
-        payload.put("log_message", encodedSerializedStringFrom((byte[]) message.getLogMessage()));
-        MessageUtils.getMetaData(message, config, Date::new).forEach(payload::put);
-        return payload.toString();
+        try {
+            JSONObject payload = new JSONObject();
+            payload.put("log_key", encodedSerializedStringFrom((byte[]) message.getLogKey()));
+            payload.put("log_message", encodedSerializedStringFrom((byte[]) message.getLogMessage()));
```

Comment:
here we are assuming that internal data is in byes[].. can we do it better?

Reply:
do we want to serialize any input data to bytes? or are we only handling specific data types?